ssh/authorisedkeys.go

```diff
@@ -15,10 +15,9 @@ import (
 	"sync"
 
 	"github.com/juju/errors"
-	"github.com/juju/loggo/v2"
+	"github.com/juju/loggo"
+	"github.com/juju/utils/v3"
```

Comment:
The utils import needs to stay where it was.

But this is the wrong branch, it needs to be v3 branch if we want to use these changes in juju 3.6
The master branch is for v4